common/extensions/api/_api_features.json

```diff
@@ -106,7 +106,8 @@
     "matches": [
       "chrome://welcome/*",
       "chrome://settings/*",
-      "chrome://newtab/*"
+      "chrome://newtab/*",
+      "chrome://report/*"
```

Comment:
`report` doesn't seem like a good choice to me because that could mean almost anything. `chrome://webcompat`?